intelmq/bots/experts/cut_from_string/expert.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+Cut from string
+"""
+from intelmq.lib.bot import Bot
+
+
+class CutFromStringExpertBot(Bot):
+    string_from_start: int = 1  # 1 - from start, 0 - from end
```

Comment:
If the valid values for these parameters are just 0 and 1 why not use a boolean value? From a user-perspective, maybe string values like `cut_from = 'start'` are better? (just thinking aloud)

Reply:
Good point.
It can be improved.

Reply:
Given that there are only two possible values, it should IMO be a boolean